plasmapy/dispersion/two_fluid_dispersion_solver.py

```diff
@@ -0,0 +1,243 @@
+__all__ = ["two_fluid_dispersion_solution"]
+
+import astropy.units as u
+import numpy as np
+
+from astropy.constants.si import c, e, k_B, m_e, m_p, mu0
+
+import plasmapy.formulary.parameters as pfp
+
+from plasmapy.utils.decorators import validate_quantities
+
+
+@validate_quantities(
+    B={"can_be_negative": False},
+    m_e={"can_be_negative": False},
+    m_i={"can_be_negative": False},
+    n={"can_be_negative": False},
+    T_e={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    T_i={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    theta={"can_be_negative": True},
+)
+def two_fluid_dispersion_solution(
+    *,
+    B: u.T,
+    k: u.m ** -1,
+    n: u.m ** -3,
+    T_e: u.K,
+    T_i: u.K,
+    gamma_e=1,
+    gamma_i=3,
+    ion="p+",
+    m_e: u.kg = m_e,
+    m_i: u.kg = m_p,
+    theta: u.deg = 45 * u.deg,
+    z=1,
+):
+    r"""
+    Return a dictionary of frequencies corresponding to the
+    solutions of dispersion relation in low frequency regime.
```

Comment:
```suggestion
    Return a dictionary of frequencies corresponding to the solutions of
    dispersion relation in low frequency regime.
```

Our current max line length is 88, so don't worry about wrapping text until you're past 80 characters.